src/main/java/com/jfrog/ide/idea/scan/ScanManager.java

```diff
@@ -76,7 +76,7 @@ public void startScan() {
             return;
         }
 
-        if (!GlobalSettings.getInstance().reloadXrayCredentials()) {
+        if (!GlobalSettings.getInstance().reloadMissingConfiguration()) {
```

Comment:
Isn't the `reloadXrayCredentials` function call needed? .
If not, let's change the error message on line 80